browser/brave_profile_prefs.cc

```diff
@@ -203,6 +204,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   brave_shields::RegisterShieldsP3AProfilePrefs(registry);
 
   brave_news::BraveNewsController::RegisterProfilePrefs(registry);
+  commands::AcceleratorPrefManager::RegisterProfilePrefs(registry);
```

Comment:
you might want to call this from `AcceleratorServiceFactory::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry)` override.